.github/pull_request_template.md

```diff
@@ -2,15 +2,47 @@
 
 ⚠️ **If your contribution is more than a few lines of code, then prior to starting to code on it please post in the issue saying you want to volunteer, then wait for a positive response.** And if there is no issue for it yet, create it first.
 
-This helps make sure (1) two people aren't working on the same thing, (2) this is something Streamlit's maintainers believe should be implemented/fixed, (3) any API, UI, or deeper architectural changes that need to be implemented have been fully thought through by Streamlit's maintainers, and (4) your time is well spent!
+This helps make sure:
+
+1. Two people aren't working on the same thing
+2. This is something Streamlit's maintainers believe should be implemented/fixed
+3. Any API, UI, or deeper architectural changes that need to be implemented have been fully thought through by Streamlit's maintainers
+4. Your time is well spent!
 
 More information in our wiki: https://github.com/streamlit/streamlit/wiki/Contributing
 
 ---
 
-**Issue:** Please include a link to the issue you're addressing. If no issue exists, create one first and then link it here.
+## 📚 Context
+
+_Please describe the project or issue background here_
+
+## 🧠 Description of Changes
```

Comment:
We might add even more categories here (or in a separate section: `What kind of change does this PR introduce?`), e.g.:

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation
- [ ] Other, please describe:

Reply:
I think this would be a helpful add! 
Going to put it into the context section, just w/o the docs option since we are starting to separate that from this repo.